app/services/edge_cache/bust_commentable.rb

```diff
@@ -0,0 +1,11 @@
+module EdgeCache
+  class BustCommentable < Bust
+    def self.call(commentable)
+      return unless commentable
+
+      EdgeCache::BustComment.call(commentable)
+      bust("#{commentable.path}/comments")
+      commentable.index_to_elasticsearch_inline
```

Comment:
as `BustComment` is never actually called outside `BustCommentable`, what if we merge these two files in `BustCommentable` ?

Reply:
It is called outside `BustCommentable`:

https://github.com/forem/forem/blob/7f066dd2036c91a7a9742b89f26b0c581be5014b/app/services/users/delete_articles.rb#L14

https://github.com/forem/forem/blob/7f066dd2036c91a7a9742b89f26b0c581be5014b/app/services/users/delete_comments.rb#L10

Both of those services call other bust methods under the old `CacheBuster`. I'll refactor them once all of the methods are "moved over".

Reply:
got it! Thanks!